src/arrays/referencing_array_and_its_elements/medium/q10/VowelNames.java

```diff
@@ -0,0 +1,30 @@
+package arrays.referencing_array_and_its_elements.medium.q10;
+
+/*
+Write a Java program that asks the user for 6 names and stores them in an array of Strings. Then, count how many of the
```

Comment:
Change this so it displays the names that start with B not just counting them, and increasing the input size.